techlibs/intel/synth_intel.cc

```diff
@@ -187,8 +187,14 @@ struct SynthIntelPass : public ScriptPass {
 		}
 
 		if (!nobram && check_label("map_bram", "(skip if -nobram)")) {
-			run("memory_bram -rules +/intel/common/brams.txt");
-			run("techmap -map +/intel/common/brams_map.v");
+                        if (family_opt == "cycloneiv" ||
+                            family_opt == "cycloneive" ||
+                            family_opt == "max10") {
+				run("memory_bram -rules +/intel/common/brams_m9k.txt");
+				run("techmap -map +/intel/common/brams_map_m9k.v");
```

Comment:
Maybe this should be `|| help_mode` and a comment (2nd arg) should be added to these two `run()`. Something like `"(if applicable for FPGA family)"`. Otherwise `help synth_intel` will not produce the desired output.

Reply:
Added this:
```
    map_bram:    (skip if -nobram)
        memory_bram -rules +/intel/common/brams_m9k.txt    (if applicable for family)
        techmap -map +/intel/common/brams_map_m9k.v    (if applicable for family)
```

When future BRAM cells are added, (e.g. M10K for Cyclone V, M20K for gen10), should there be equivalent comments for those, or should we rework the comments for that?